akka-bench-jmh/src/main/scala/akka/stream/SourceRefBenchmark.scala

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (C) 2014-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream
+
+import java.util.concurrent.{ Semaphore, TimeUnit }
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.stream.impl.fusing.GraphStages
+import akka.stream.scaladsl._
+import com.typesafe.config.ConfigFactory
+import org.openjdk.jmh.annotations._
+import org.reactivestreams._
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.util.Success
+
+/*
+   Just a brief reference run (3.1 GHz Intel Core i7, MacBook Pro late 2017):
+   [info] SourceRefBenchmark.flow_map_100k_elements  thrpt   10  724650.336 ± 233643.256  ops/s
```

Comment:
copied from somewhere?

Reply:
Pasted example result from my macbook, should I remove?

Reply:
sorry, `flow_map_100k_elements` looked like it was another test

Reply:
Oh I see, I left the method name, renaming